analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.cs

```diff
@@ -195,7 +203,6 @@ public static class Extensions
         [DataRow("var dummy = this.Property;")]
         [DataRow("SampleProperty.InstanceMethod();")]
         [DataRow("this.SampleProperty.InstanceMethod();")]
-        [DataRow("this.SampleProperty?.InstanceMethod();")]
```

Comment:
Educational: why this scenario was removed from the test? My understanding is that, unlike all other scenarios listed here, this one branches the state into two: one where `SampleProperty` is `null` and one where it isn't. 
However, I don't see how this would affect the fact that `ObjectField` and `StaticObjectField` should be null.